R/archive.R

```diff
@@ -0,0 +1,184 @@
+#' Archive SPI-Birds raw data
+#'
+#' @param data_folder Path. Folder containing all current SPI-Birds data (N drive)
+#' @param PopID Character string. 3-4 letter code for population.
+#' Should be the same as the folder name where data are saved.
```

Comment:
I partially agree with Stefan here. For the folder paths, it should probably be "Owner" or "DataOwner" since that is how the folders are named in the N-drive. However, I am not sure if we may have cases in which the folder is named using the data owner's code, but separate primary data files within are names using the PopID. If that was the case, we may have to do some re-naming...or find another workaround.

Reply:
I think you're potentially both right here. This should be DataOwner because we're archiving at the folder level (which is arranged by data owner), but Chloe is right that there could be cases where there are multiple files from different PopIDs in the folder of a given data owner. I think the best practice in that case would be to name the files something like `DataOwner_PrimaryData_PopID`, that way they will still get archived properly.

Reply:
Ah, I understand now. The workaround seems alright then!